backend/data/models.json

```diff
@@ -0,0 +1,127 @@
+[
+    {
+        "id": "tiny",
+        "name": "Tiny",
+        "languages": [
+            "auto",
```

Comment:
what is this list and where does it come from?

Reply:
These are the alignable languages, right? These have nothing to do with the whisper languages?

Reply:
This is the overlap of the languages that whisper understands and the one we can align.